### ISSUE TITLE
server connection error when trying to download anything

### ISSUE THREAD

**Issue by shiftypickle88** [Ibrahim778/BetterHomebrewBrowser#56]
i am going to go through everything that i did in the hour or two before the issue arose

i installed BHBB, then immediately opened it and downloaded the nazi zombies port and i believe that when i went  to run it, it informed me that i needed the LibShacc compiler files installed so i FTP'd the PIB installer over from my phone and that went perfectly normal. i played a couple games of zombies and then went back to BHBB to look around some more and the next thing i tried to download it failed and gave me a "Couldn't connect to server NW-8942-3" error. this happens no matter what i attempt to download. i deleted and reinstalled iTLS-Enso and that did not change anything.

**Comment by shiftypickle88** [comment]
ok i have a little bit of an update, so initially this was all while browsing and downloading from the VitaDB server.

![20241231_065210](https://github.com/user-attachments/assets/b7d80f80-086e-48ab-a853-1c15e38797a9)
 i switched to CBPS DB and was able to download a game file (contract demon). i then received the error in the attached image, BHBB then force closed. i believe this error is identical to the error i was receiving earlier today (first time botting my device in months) when attempting to download from the OG homebrew browser which is what motivated me to install BHBB in the first place

i am getting some strange behavior now that i have switched to the CBPS DB server, i am getting "error loading description C0-11274-5" every time i select a download. although i can download the file successfully on CBPS DB

**Comment by Ibrahim778** [comment]
I currently have my hands tied with college applications. I will work on this whenever I can but it probably wont be soon.
I will note though that CBPS DB is old and hasn't been updated in years or so, expect bugs and outdated links when using it.

**Comment by shiftypickle88** [comment]
Yeah i'm mostly concerned with the complete inability to download from
vitadb

**Comment by Ibrahim778** [comment]
The issue with VitaDB was on the VitaDB side. I've talked to rinne and we've sorted it out, it should be working again.
As for CBPS-DB I can't fix it right now because there's something else going wrong with my version of the app and I don't have enough time to fix it at the moment.

**Comment by shiftypickle88** [comment]
No worries, I appreciate your time. Good luck with your college applications
